src/cards/cover-card/cover-card.ts

```diff
@@ -229,20 +255,36 @@ export class CoverCard extends MushroomBaseCard implements LovelaceCard {
                         .fill=${layout !== "horizontal"}
                     />
                 `;
-            case "position_control":
-                const color = getStateColor(entity as CoverEntity);
+            case "position_control": {
+                    const color = getStateColor(entity as CoverEntity);
+                    const sliderStyle = {};
+                    sliderStyle["--slider-color"] = `rgb(${color})`;
+                    sliderStyle["--slider-bg-color"] = `rgba(${color}, 0.2)`;
+
+                    return html`
+                        <mushroom-cover-position-control
+                            .hass=${this.hass}
+                            .entity=${entity}
+                            @current-change=${this.onCurrentPositionChange}
+                            style=${styleMap(sliderStyle)}
+                        />
+                    `;
+                }
+            case "tilt_position_control": {
+                const color = getTiltStateColor(entity as CoverEntity);
```

Comment:
```suggestion
                const color = getStateColor(entity as CoverEntity);
```
The same color should be used for the control. No need to have a separate function and new css variables.

Reply:
I removed `tiltStateColor`. Let's see how people use it, and maybe there isn't any difference in perception ;)